GroupBundle/EventListener/CreateGroupListener.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace OpenOrchestra\GroupBundle\EventListener;
+
+use Doctrine\ODM\MongoDB\Event\LifecycleEventArgs;
+use OpenOrchestra\BackofficeBundle\Model\GroupInterface;
+use OpenOrchestra\ModelInterface\Model\SiteInterface;
+
+/**
+ * Class CreateGroupListener
+ */
+class CreateGroupListener extends AbstractNodeGroupRoleListener
```

Comment:
I think that the name should be related to what the listener does and not to what the listener listen to